packages/i18n/data/core.json

```diff
@@ -37,23 +39,27 @@
   "Login.sso.proceed": "Proceed to sign in",
   "Login.sso.title": "Sign in to your Organization",
   "LoginForm.email": "Email",
-  "LoginForm.forgotPassword": "Forgot password?",
+  "LoginForm.forgotPassword": "Forgot password",
   "LoginForm.forgotPasswordTitle": "We are redirecting you to the Forgot Password form in {countdown} seconds.",
   "LoginForm.InvalidCredentials": "Sorry, but the login details entered below are not in our system. Please enter your correct email address and password below.",
   "LoginForm.invalidEmail": "Please enter a valid Email address.",
   "LoginForm.logout": "We have logged you out of the Merchant Center. If you would like to log back in, please fill out the form below.",
   "LoginForm.noProjectsLink": "set up a project in the current Merchant Center",
   "LoginForm.noProjectsTitle": "Sorry, but you do not have a project associated to your account and we are therefore unable to log you in. Please {link} before trying again.",
   "LoginForm.password": "Password",
-  "LoginForm.signin": "Sign in",
-  "LoginForm.title": "Sign in to your account",
+  "LoginForm.signin": "Log in",
```

Comment:
login is the noun, "log in" is the verb. I think we want the noun here, since "Password" is also a noun.

Reply:
Agreed